ext/opcache/tests/preload_009.phpt

```diff
@@ -13,6 +13,6 @@ var_dump(trait_exists('T'));
 var_dump(class_exists('Foo'));
 ?>
 --EXPECTF--
-Warning: Can't preload class Foo with unresolved initializer for constant C in %spreload_undef_const_2.inc on line 8
+Warning: Use of undefined constant UNDEF - assumed 'UNDEF' (this will throw an Error in a future version of PHP) in Unknown on line 0
```

Comment:
This is a questionable decision. It may be better to throw error with a hint to try preloading through opcache_compile_file(). Anyway, this may be fixed later.